feature-libs/organization/administration/components/unit/list/unit-list.component.ts

```diff
@@ -13,7 +14,12 @@ import { UnitTreeService } from '../services/unit-tree.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class UnitListComponent {
-  constructor(protected unitTreeService: UnitTreeService) {}
+  constructor(
+    protected unitTreeService: UnitTreeService,
+    protected orgUnitService?: OrgUnitService
+  ) {}
+
+  isUpdatingUnitAllowed = this.orgUnitService?.isUpdatingUnitAllowed();
```

Comment:
this may be left as `undefined` better to add  a conditional here and set `isUpdatingUnitAllowed` as false if `this.orgUnitService` is not defined.

Reply:
done